public/javascripts/common/wrapperFrame.js

```diff
@@ -0,0 +1,53 @@
+const iframe = document.getElementById('wrapperFrame');
+const testUserBanner = document.querySelector('.test-server-banner');
+
+function iframeURLChange(callback) {
+    var unloadHandler = function () {
+        iframe.style.display = "none";
+        setTimeout(() => {
+            callback(iframe.contentWindow.location.href);
+        }, 0)
+    };
+
+    function attachUnload() {
+        iframe.contentWindow.removeEventListener("unload", unloadHandler);
+        iframe.contentWindow.addEventListener("unload", unloadHandler);
+    }
+
+    iframe.addEventListener("load", attachUnload);
+    attachUnload();
+}
+
+function scaleIframeContent() {
+    iframe.contentWindow.devicePixelRatio = 1;
+    const iframeDocument = iframe.contentDocument || iframe.contentWindow.document;
+    const contentElement = iframeDocument.querySelector('.tool-ui');
+
+    if (contentElement) {
+        const iframeWidth = window.innerWidth;
+        const iframeHeight = window.innerHeight - 70 - (testUserBanner ? testUserBanner.clientHeight : 0);
+        const contentWidth = contentElement.clientWidth;
+        const contentHeight = contentElement.clientHeight + 100; // Add 100px for padding purposes.
```

Comment:
If I wanted to change the amount of padding, is this the only place where that would need to be changed, or do edits need to be made in multiple places?

Reply:
```js
        const contentHeight = contentElement.clientHeight + 100; // Add 100px for padding purposes.
```

This should be the only place you'd need to change.